roles/science/tasks/mrtrix-dev.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Create directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  loop:
+    - /opt/quarantine/software/mrtrix3/dev
```

Comment:
Lets use the group_vars method I've used elsewhere to fix a tag/commit version for reproducitibility, we can then update as needed.